### PR TITLE
Feat/nexus [Do not merge]

### PR DESCRIPTION
**Pull request by joepegler**
## PR-Codex overview
This PR focuses on enhancing documentation for the `Biconomy` SDK and `Nexus SDK`, including quickstart guides and code examples, as well as making minor updates to the sidebar and `package.json` dependencies.

### Detailed summary
- Added a quickstart guide for `React` in `docs/pages/quickstart-react.mdx`.
- Added a quickstart guide for `Nexus SDK` in `docs/pages/nexus/quickstart.mdx`.
- Updated sidebar links in `vocs.config.ts` for clarity.
- Modified `package.json` to include `nexus` as a dependency.
- Introduced a new code example in `docs/pages/nexus/overview.mdx` demonstrating `Nexus SDK` usage.
- Removed outdated code snippets from `docs/pages/nexus/overview.mdx`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by AUCrypto-bit** [comment]
Noice

File: docs/pages/nexus/overview.mdx [hunk 1]
```diff
@@ -6,6 +6,23 @@ title: Nexus SDK Overview
 # What is Nexus?
 Nexus represents the next generation of smart account technology, built on the foundation of ERC-7579. It's a significant evolution from our previous [Biconomy Smart Account v2](https://docs.biconomy.io/account), offering:
 
+```tsx twoslash
+import { createNexusClient } from "nexus/packages/sdk";
+import { privateKeyToAccount } from "viem/accounts";
+import { baseSepolia } from "viem/chains";
+import { http } from "viem";
+
+const privateKey = "0x...";
```
Comment (AUCrypto-bit) [inline docs/pages/nexus/overview.mdx:15]: 0xfF0d2Cf1E52d2C2865Eba2D3d3BB5e952E2B5C68

File: docs/pages/nexus/overview.mdx [hunk 2]
```diff
@@ -6,6 +6,23 @@ title: Nexus SDK Overview
 # What is Nexus?
 Nexus represents the next generation of smart account technology, built on the foundation of ERC-7579. It's a significant evolution from our previous [Biconomy Smart Account v2](https://docs.biconomy.io/account), offering:
 
+```tsx twoslash
+import { createNexusClient } from "nexus/packages/sdk";
+import { privateKeyToAccount } from "viem/accounts";
+import { baseSepolia } from "viem/chains";
+import { http } from "viem";
```
Comment (AUCrypto-bit) [inline docs/pages/nexus/overview.mdx:13]: +import USDC {https://exchange.coinjar.com}from "base"main and as many other networks as you can and relay back to my wallet>

**Review by AUCrypto-bit — CHANGES_REQUESTED** [review]
Can someone set this up for me I been at it for days and only seem to be getting more confused and I might melt into a puddle off goo at this rate trying me best not to turn into a supervillain

**Comment by himanshugarg06** [comment]
@joepegler should we close this PR?

**Pull request closed without merging**